spec/support/corpus.rb

```diff
@@ -226,6 +245,60 @@ def system(arguments)
         end
       end
 
+      # Mapping of files which we expect to cause errors during mutation generation
+      class ErrorWhitelist
+        class UnnecessaryExpectation < StandardError
+          MESSAGE = 'Expected to encounter %s while mutating "%s"'.freeze
+
+          def initialize(error, path)
```

Comment:
If you wanted to you could use `*error_info` here, and then do `super(MESSAGE % error_info)`. You're not really using any of the values on their own, and really just proxying them to `#%`.